src/components/external-dns/dev.tsx

```diff
@@ -0,0 +1,110 @@
+import { ExternalDNSList } from '.';
+
+const testData: Array<Parameters<typeof ExternalDNSList>[0]> = [
```

Comment:
use ComponentProps<> instead of Parameters<>:
```sugestion
const testData: Array<ComponentProps<typeof ExternalDNSList>> = [
```

Reply:
fixed